src/sentry/static/sentry/app/views/eventsV2/index.tsx

```diff
@@ -80,29 +98,16 @@ class EventsV2 extends React.Component<Props> {
     return <LinkList>{list}</LinkList>;
   }
 
-  getEventViewName = (): Array<string> => {
-    const {location} = this.props;
-
-    const name = getFirstQueryString(location.query, 'name');
-
-    if (typeof name === 'string' && String(name).trim().length > 0) {
-      return [t('Discover'), String(name).trim()];
-    }
-
-    return [t('Discover')];
-  };
-
   render() {
     const {organization, location, router} = this.props;
+
     const eventSlug = getFirstQueryString(location.query, 'eventSlug');
     const eventView = EventView.fromLocation(location);
+    const queryName = getFirstQueryString(location.query, 'name', t('Untitled query'));
```

Comment:
@leedongwei The `name` query string should be already derived from `EventView.fromLocation(location)`.

The name should be accessible from `eventView.name`.

Reply:
Resolved with the latest force-push.